src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -399,6 +399,36 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_ExtendedQueryTag_TagPath ON dbo.ExtendedQuer
     TagPath
 )
 
+/*************************************************************
+    Extended Query Tag Errors Table
+    Stores errors from Extended Query Tag operations
+    TagKey and Watermark is Primary Key
+**************************************************************/
+IF NOT EXISTS (
```

Comment:
Peng/Will, please follow 3.sql pattern for re-runnability, it is less code and more readable.